build/commands/lib/util.js

```diff
@@ -1104,6 +1110,54 @@ const util = {
         },
       )
       removeUnlistedAndroidResources(braveOverwrittenFiles)
+
+      console.log('Remove unlisted overridden pngs')
```

Comment:
is this just meant to be a cleanup after they are already copied?

Reply:
no this is to remove files physically after excluding them like at 
https://github.com/brave/brave-core/pull/29688/files#diff-35f5c0b2c4fce179586fc373b8dd61bd966d81388d3f24ecca28635b6cdb67d6R970

otherwise there are errors like `found file not listed at any target`

Reply:
so these are upstream sources? This seems problematic to me, I already want to find a better alternative to just overwriting files like this, but deleting them seems even worse. I worry this is going to cause issues when switching between branches among other concerns